pkg/nodehandler/controller.go

```diff
@@ -229,9 +231,47 @@ func addNodeRemovalLabel(csibmnode *nodecrd.Node) {
 	if csibmnode.Labels == nil {
 		csibmnode.Labels = map[string]string{}
 	}
-	csibmnode.Labels[nodeRemovalTaintKey] = nodeRemovalTaintValue
+	csibmnode.Labels[rTaint.Key] = rTaint.Value
 }
 
 func deleteNodeRemovalLabel(csibmnode *nodecrd.Node) {
-	delete(csibmnode.Labels, nodeRemovalTaintKey)
+	delete(csibmnode.Labels, rTaint.Key)
+}
+
+func (c *Controller) handleNodeMaintenance(ctx context.Context, nodes []corev1.Node) error {
+	for i, node := range nodes {
+		if hasTaint(&nodes[i], mTaint) {
+			if err := c.deleteCSIPods(ctx, node.Name); err != nil {
+				return err
```

Comment:
Could we try to delete all pods before returning error?

Reply:
Did you mean we need to ignore errors for some pods until loop is over?

Reply:
By spec, if some pods can't be removed, need to take off taint. And killed pods will be scheduled again.

Reply:
We can have network or timeout issue

Reply:
Also in the beginning, if  `handleNodeRemoval` has error, `handleNodeMaintenance` never happens

Reply:
Done